harpy/downsample.py

```
@@ -67,37 +67,37 @@ def downsample(input, invalid, output_dir, prefix, downsample, random_seed, hpc,
         "downsample" :  downsample,
         "invalid_proportion" : invalid,       
         **({"random_seed" : random_seed} if random_seed else {}),
-        "workflow_call" : command.rstrip(),
+        "snakemake_command" : command.rstrip(),
         "inputs": [Path(i).resolve().as_posix() for i in input]
-        }
-    with open(os.path.join(workflowdir, 'config.yaml'), "w", encoding="utf-8") as config:
-        yaml.dump(configs, config, default_flow_style= False, sort_keys=False, width=float('inf'))
+    }
 
+    write_workflow_config(conrigs, workflowdir)
```

Ruff (0.8.2)

Undefined name `conrigs`

(F821)